python/extension/CMakeLists.txt

```diff
@@ -138,6 +139,9 @@ add_mlir_python_modules(CUDAQuantumPythonModules
     CUDAQuantumPythonCAPI
   )
 
+set_target_properties(CUDAQuantumPythonModules.extension._quakeDialects.dso PROPERTIES CXX_VISIBILITY_PRESET "default")
+target_compile_options(CUDAQuantumPythonModules.extension._quakeDialects.dso PRIVATE -Wno-attributes)                     
```

Comment:
I don't think the -Wno-attributes is needed, might have been left around from my prototyping

Reply:
Should we remove both of these lines?